### PR TITLE
Code coverage is skipping Azure core

### PR DESCRIPTION
**Pull request by praveenkuttappan**
1. Coverage is not combining all source paths so it skips installed azure core modules
2. coverage is updated to run for current package
3. Fixed misc fixes to refer correct arg parser

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: scripts/devops_tasks/setup_execute_tests.py [hunk 1]
```diff
@@ -33,6 +34,23 @@
...
+    # find tox.ini file. tox.ini is used to combine coverage paths to generate formatted report
+    tox_ini_file = os.path.join(root_dir, "eng", "tox", "tox.ini")
+    config_file_flag = "--rcfile={}".format(tox_ini_file)
+
+    if os.path.isfile(tox_ini_file):
+        # for every individual coverage file, run coverage combine to combine path
+        for coverage_file in coverage_files:
+            cov_cmd_array = [sys.executable, "-m", "coverage", "combine"]
+            # tox.ini file has coverage paths to combine
+            # Pas tox.ini as coverage config file
```
Comment (scbedd) [inline scripts/devops_tasks/setup_execute_tests.py]: nit "pass"

**Pull request merged**